src/main/madara-app.ts

```diff
@@ -190,7 +190,93 @@ export async function startApp(window: BrowserWindow, appId: string) {
   if (appConfig.appType === 'binary') {
     await startAppBinary(window, appConfig);
   } else if (appConfig.appType === 'docker') {
-    await startAppDocker(window, appConfig);
```

Comment:
we shouldn't be commenting this out, i guess you did it for testing but still adding the comment so we don't forget